e2e/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "openmct-e2e",
+  "version": "4.0.0-next",
```

Comment:
this should always match `openmct's package.json`. Is there an easy way to manage that based on monorepo tooling?